creator-node/src/services/stateMachineManager/stateMonitoring/findSyncRequests.jobProcessor.js

```diff
@@ -51,100 +51,87 @@ module.exports = async function ({
   const unhealthyPeersSet = new Set(unhealthyPeers || [])
```

Comment:
notable changes here:

findSyncsForUser now returns outcomes with syncMode in addition to result
instead of SyncPrimaryFromSecondary having its own log + result type (see prometheus.constants changes), it is now a different label, since all the other result types also apply to that sync mode
most other changes follow on from above, including refactor + simplification of metric recording + logging code